pkg/volume/plugins.go

```diff
@@ -164,6 +164,11 @@ type VolumeHost interface {
 	// implemented in terms of the "emptyDir" volume.
 	NewWrapperMounter(volName string, spec Spec, pod *api.Pod, opts VolumeOptions) (Mounter, error)
 
+	// NewWrapperAttacher creats an Attacher which wraps one belonging to
+	// another volume. This is used to create Attachers for plugins which
+	// wrap other plugins such as the persistentClaimPlugin
+	NewWrapperAttacher(volName string, spec Spec, pod *api.Pod) (Attacher, error)
```

Comment:
The name maybe misleading here but this function is called after a deference to create an Attacher of the underlying PV. This is the mechanism by which PVCs are mounted currently.. So I just followed that pattern to try to avoid exporting too many kubelet functions... I think there is a better way to do it though.. I'll clean this up and address other feedback